### PR TITLE
SP translations

### PR DESCRIPTION
**Pull request by carolinadilello**
**Description**

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/public/locales/es/translation.json [hunk 1]
```diff
@@ -539,34 +539,34 @@
...
-        "SENSOR_BRAND": "MISSING - Es - Invalid brand name, must be between 1 and 100 characters.",
-        "SENSOR_MODEL": "MISSING - Es - Invalid model name, must be between 1 and 100 characters.",
-        "SENSOR_HARDWARE_VERSION": "MISSING - Es - Invalid hardware version name, must be between 1 and 100 characters."
+        "FILE_ROW_LIMIT_EXCEEDED": "Límite de fila de archivos superado. El número máximo de sensores que se pueden cargar desde un archivo es 100.",
+        "MISSING_COLUMNS": "Se requieren columnas/faltan.",
+        "EXTERNAL_ID": "Identificación externa no válida, debe tener entre 1 y 20 caracteres.",
+        "SENSOR_NAME": "Nombre del sensor no válido, debe estar entre 1 y 100 caracteres.",
+        "SENSOR_LATITUDE": "Valor de latitud no válido, debe estar entre -90 y 90. y menos de 10 decimales.",
+        "SENSOR_LONGITUDE": "Valor de longitud no válido, debe estar entre -180 y 180. y menos de 10 decimales.",
+        "SENSOR_READING_TYPES": "Tipo de lectura no válido detectado, los valores válidos incluyen: soil_water_content, soil_water_potential, temperature.",
```
Comment (SayakaOno) [inline packages/webapp/public/locales/es/translation.json:548]: Could you check this translation? {{ reading_types }} needs to be included.
English: `"Invalid reading type detected: {{ reading_types }}. Valid values include: soil_water_content, soil_water_potential, temperature."`

File: packages/webapp/public/locales/es/translation.json [hunk 2]
```diff
@@ -1711,102 +1711,99 @@
...
     "TEMPERATURE_READINGS_OF_SENSOR": {
-      "TITLE": "MISSING",
-      "SUBTITLE": "MISSING",
-      "Y_AXIS_LABEL": "MISSING",
-      "WEATHER_STATION": "MISSING",
-      "NO_DATA": "MISSING",
-      "AMBIENT_TEMPERATURE_FOR": "MISSING",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "MISSING"
+      "TITLE": "Temperatura del suelo",
+      "SUBTITLE": "Temperatura ambiente alta y baja de hoy: {{alta}}{{tempUnit}} / {{baja}}{{tempUnit}}",
```
Comment (SayakaOno) [inline packages/webapp/public/locales/es/translation.json:1772]: Please keep the texts in `{{ }}` (high and low) :)
English: `"Today’s ambient high and low temperature: {{high}}{{tempUnit}} / {{low}}{{tempUnit}}"`

File: packages/webapp/public/locales/es/translation.json [hunk 3]
```diff
@@ -1711,102 +1711,99 @@
...
+      "Y_AXIS_LABEL": "Temperatura en {{tempUnit}}",
+      "WEATHER_STATION": "Estación meteorológica: {{Estación meteorológica}}",
```
Comment (SayakaOno) [inline packages/webapp/public/locales/es/translation.json:1774]: Please keep `{{weatherStationLocation}}`

**Review by SayakaOno — CHANGES_REQUESTED** [review]

File: packages/webapp/public/locales/es/translation.json (hunk 2 above)
Comment (carolinadilello) [reply packages/webapp/public/locales/es/translation.json:1772]: This is translated. High is "alta" and Low is "baja". Do I also need to translate tempUnit?

**Review by SayakaOno — DISMISSED** [review]

**Review by SayakaOno — APPROVED** [review]

**Pull request merged**